test/k8sT/Services.go

```diff
@@ -1252,6 +1253,7 @@ var _ = Describe("K8sServicesTest", func() {
 					})
 
 					testDSR(64001)
+					testNodePort(true, false, false) // no need to test from outside, as testDSR did it
```

Comment:
The previous tests also cleared the CT tables on the node after `testNodePort()`, is that handled here somehow or not necessary?

Reply:
As far as I can see, it's not necessary. I haven't seen a failure due to potentially stale CT entries.

Reply:
Good enough for me, we can keep an eye out in case this concern is warranted.